docs/ibc/apps.md

```diff
@@ -160,6 +160,36 @@ Application modules are expected to verify versioning used during the channel ha
 * `ChanOpenTry` callback should verify that the `MsgChanOpenTry.Version` is valid and that `MsgChanOpenTry.CounterpartyVersion` is valid.
 * `ChanOpenAck` callback should verify that the `MsgChanOpenAck.CounterpartyVersion` is valid and supported.
 
+IBC expects modules to implement the function `NegotiateAppVersion` from the application module
+interface. This function performs application version negotiation and returns the negotiated version.
```

Comment:
Small nit here, [usually "method" is used](https://go.dev/doc/effective_go#interface_methods) when referring to a function that exists on an interface. And changed "application module" to [`IBCModule`](https://github.com/cosmos/ibc-go/blob/main/modules/core/05-port/types/module.go#L13)

```suggestion
IBC expects application modules to implement the `NegotiateAppVersion` method from the `IBCModule`
interface. This method performs application version negotiation and returns the negotiated version.
```